qiita_pet/support_files/doc/source/tutorials/account-creation.rst

```diff
@@ -1,64 +0,0 @@
-.. _account-creation:
```

Comment:
I think we need to improve the documentation that is in the CMI workshop ... it could be a copy/paste of this.